doom-modeline-core.el

```diff
@@ -474,8 +474,26 @@ in the given order."
   "4.2.0")
 
 (defcustom doom-modeline-check-simple-format t
-  "If non-nil, only display one number for check information if applicable."
-  :type 'boolean
+  "If non-nil, only display one number for check information if applicable.
+If set to icons, display check information with icons."
+  :type '(choice boolean
+                 (const :tag "simple icons" icons))
+  :group 'doom-modeline)
+
+(defcustom doom-modeline-check-warning-icon
```

Comment:
I'd like`doom-modeline-check-warning-icon` and `doom-modeline-check-note-icon` are constants rather than options.
And how about `doom-modeline-check-error-icon`?

Reply:
> I'd like`doom-modeline-check-warning-icon` and `doom-modeline-check-note-icon` are constants rather than options. 

I don't think this can be a good idea, that would not allow to user change their icons.

> Otherwise the new options should be added to README as well. And how about `doom-modeline-check-error-icon`?

Thank you i've added them.